llama_index/callbacks/token_counting.py

```diff
@@ -50,7 +50,7 @@ def get_llm_token_counts(
             response_tokens = 0
 
             if response is not None and response.raw is not None:
-                usage = dict(response.raw.get("usage"))  # type: ignore
+                usage = response.raw.get("usage")
```

Comment:
I think this breaks openai compatibility 

Ideally, we check if usage is there, if it is there, convert it to dict and pull the info

Reply:
Good suggestion, will update tomorrow. 
I tested it successfully with openai, though :)